src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java

```diff
@@ -453,7 +477,8 @@ protected void copyProxyCookie(HttpServletRequest servletRequest,
       Cookie servletCookie = new Cookie(proxyCookieName, cookie.getValue());
       servletCookie.setComment(cookie.getComment());
       servletCookie.setMaxAge((int) cookie.getMaxAge());
-      servletCookie.setPath(path); //set to the path of the proxy servlet
+      servletCookie.setPath(cookie.getPath());
```

Comment:
I fix the cookie issue with adding the init param "useOriginalCookiePath" and restore getRealCookie method.